tests/unit/src/main/scala/tests/BaseAmmoniteSuite.scala

```diff
@@ -678,7 +678,7 @@ abstract class BaseAmmoniteSuite(scalaVersion: String)
     val expected =
       """|
          |
-         |<<import>>/*keyword*/ <<util>>/*namespace*/.{<<Failure>>/*class*/ <<=>>>/*operator*/ <<NotGood>>/*class*/}
+         |<<import>>/*keyword*/ <<scala>>/*namespace*/.<<util>>/*namespace*/.{<<Failure>>/*class*/ <<=>>>/*operator*/ <<NotGood>>/*class*/}
```

Comment:
this test was flaky: Presentation compiler took `util` as `ammonite.util` not `scala.util`, and `ammonite.util` doesn't have member `Failure`